serious_stuff/lib/screens/MapPage.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+Completer<GoogleMapController> _controller = Completer();
+
+const LatLng _center = const LatLng(49.2780947,-122.9373926);
```

Comment:
Do we want to keep the coordinates as constants somewhere? If we don't need them again it probably doesn't matter much then

Reply:
Also please have spaces after operators like `+`, `-` or `,`

Reply:
this constant is the center of sfu burnaby so well need it again, so we should put it somewhere